src/css/main.scss

```diff
@@ -106,13 +106,44 @@ $blue-epsilon: #bce7f7;
 
 .form-group {
   &.required {
-  .col-form-label:after {
-    content: "*";
-    color: red;
+    .col-form-label:after {
+      content: "*";
+      color: red;
+    }
   }
+}
+
+.create-form {
+  border-radius: 5px;
```

Comment:
If there is no horizontal padding here, then rounded corners are looking a bit odd (also other steps don't have rounded corners on outside containers).